arviz/plots/autocorrplot.py

```diff
@@ -23,37 +23,38 @@ def plot_autocorr(
     backend_kwargs=None,
     show=None,
 ):
-    """Bar plot of the autocorrelation function for a sequence of data.
+    """Bar plot of the autocorrelation function (ACF) for a sequence of data.
 
-    Useful in particular for posteriors from MCMC samples which may display correlation.
+    The ACF plots are helpful as a convergence diagnostic for posteriors from MCMC 
+    samples which may display correlation. The ACF is efficiently computed using a 
+    Fast Fourier Transform for every lag for the input array.
 
     Parameters
     ----------
-    data : InferenceData
+    data : obj
         Any object that can be converted to an :class:`arviz.InferenceData` object
         refer to documentation of :func:`arviz.convert_to_dataset` for details
     var_names : list of str, optional
-        Variables to be plotted, if None all variables are plotted. Prefix the
-        variables by ``~`` when you want to exclude them from the plot. Vector-value
-        stochastics are handled automatically.
-    filter_vars : {None, "like", "regex"},  default=None
-        If `None` (default), interpret var_names as the real variables names. If "like",
-        interpret var_names as substrings of the real variables names. If "regex",
-        interpret var_names as regular expressions on the real variables names. A la
-        ``pandas.filter``.
+        Variables to be plotted. Prefix the variables by ``~`` when you want to exclude
+        them from the plot. See the :ref:`this section <common_var_names>` for usage examples.
+    filter_vars : {None, "like", "regex"},  default None
+        If `None` (default), interpret `var_names` as the real variables names. If "like",
+        interpret `var_names` as substrings of the real variables names. If "regex",
+        interpret `var_names` as regular expressions on the real variables names. See
+        the :ref:`this section <common_filter_vars>` for usage examples.
     max_lag : int, optional
-        Maximum lag to calculate autocorrelation. Defaults to 100 or num draws,
-        whichever is smaller.
-    combined : bool, default=False
+        Maximum lag to calculate autocorrelation. By Default, the plot displays the 
+        first 100 lag or the total number of draws, whichever is smaller.
+    combined : bool, default False
         Flag for combining multiple chains into a single chain. If False, chains will be
         plotted separately.
-    grid : tuple
+    grid : tuple, optional
         Number of rows and columns. Defaults to None, the rows and columns are
         automatically inferred.
-    figsize : (float, float), optional
+    figsize : tuple, optional
```

Comment:
The float, float formatting is correct, no need to change it. It is a common convention that the parentheses indicate it is a tuple and this formatting allows indicating the types of each element and the number of elements. Ref: https://matplotlib.org/stable/devel/documenting_mpl.html#parameter-type-descriptions

Reply:
We used the `tuple` in plot_ess() function, lets leave this one as a standard